uqcsbot/base.py

```diff
@@ -139,8 +139,8 @@ def api(self):
         return APIWrapper(self.client)
 
     @property
-    def async(self):
-        return AsyncBotWrapper(self.client)
+    def as_async(self):
+        return AsyncBotWrapper(self)
```

Comment:
why the swap from client to self?

Reply:
It's a bot wrapper, not a client wrapper. Having `self.client` there was always a bug.